docs/BUILD.bazel

```diff
@@ -1,50 +1,48 @@
 load("@io_bazel_stardoc//stardoc:stardoc.bzl", "stardoc")
-load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
 
 package(default_visibility = ["//visibility:private"])
 
-_RULES = [
+# Targets to document.
+# Must have a coressponding `bzl_library` target in `//rules:BUILD.bazel`.
+_DOC_SRCS = [
     "app_clip",
     "app",
-    "apple_patched",
```

Comment:
This change looks orthogonal to the change

Reply:
Missed this one

I'll work on supporting this with new stardoc version and update the PR

Reply:
Fixed this up, its now included again